### ISSUE TITLE
Bug located in q.assign_source_to_objectface

### ISSUE THREAD

**Issue by ToInoJo** [ansys/pyaedt#3035]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

Axisdir is ignored. 
Assign sink works properly under same conditions.

### Steps To Reproduce

Basically make a box and try to assign a source, like:
q.modeler.create_box([y-delta*0.5,x-delta*0.5,0.3], [delta,delta,0.01], f"Net_{net_number}_start", matname="copper")
q.assign_source_to_objectface(f"Net_{net_number}_start", axisdir=q.AxisDir.ZPos, source_name=f"Net_{net_number}_source")

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

absl-py==1.4.0
astunparse==1.6.3
cachetools==5.3.0
certifi==2023.5.7
cffi==1.15.1
charset-normalizer==3.1.0
clipboard==0.0.4
clr-loader==0.2.5
contourpy==1.0.7
cycler==0.11.0
dearpygui==1.9.1
docutils==0.20
flatbuffers==23.3.3
fonttools==4.39.3
gast==0.4.0
google-auth==2.17.3
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
grpcio==1.54.0
h5py==3.8.0
idna==3.4
jax==0.4.8
keras==2.12.0
Kivy==2.1.0
kivy-deps.angle==0.3.3
kivy-deps.glew==0.3.1
kivy-deps.sdl2==0.4.5
Kivy-Garden==0.1.5
kiwisolver==1.4.4
libclang==16.0.0
Markdown==3.4.3
MarkupSafe==2.1.2
matplotlib==3.7.1
ml-dtypes==0.1.0
mypy==1.2.0
mypy-extensions==1.0.0
numpy==1.23.5
oauthlib==3.2.2
opt-einsum==3.3.0
packaging==23.1
Pillow==9.5.0
plumbum==1.8.1
protobuf==4.23.0
psutil==5.9.5
pyaedt==0.6.74
pyasn1==0.5.0
pyasn1-modules==0.3.0
pycparser==2.21
Pygments==2.15.1
PyLTSpice==4.0.2
pyparsing==3.0.9
pyperclip==1.8.2
pypiwin32==223
python-dateutil==2.8.2
pythonnet==3.0.1
pywin32==306
requests==2.30.0
requests-oauthlib==1.3.1
rpyc==5.3.1
rsa==4.9
scipy==1.10.1
six==1.16.0
tensorboard==2.12.3
tensorboard-data-server==0.7.0
tensorflow==2.12.0
tensorflow-estimator==2.12.0
tensorflow-intel==2.12.0
tensorflow-io-gcs-filesystem==0.31.0
termcolor==2.3.0
tomli==2.0.1
typing_extensions==4.5.0
urllib3==2.0.2
Werkzeug==2.3.4
wrapt==1.14.1

**Comment by Samuelopez-ansys** [comment]
Hi @ToInoJo,

This method was deprecated in version 0.6.70, you can see it in the documentation or in the pyaedt docstring:

"""Generate a source on a face of an object.

        **.. deprecated:: 0.6.70
           Use :func:`source` method instead.**

The face ID is selected based on the axis direction. It is the face that
has the maximum/minimum in this axis direction.


Please try source method, if it works please close this Issue:

https://aedt.docs.pyansys.com/version/dev/API/_autosummary/pyaedt.q3d.Q3d.source.html#source
